books/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+
+
+class Book(models.Model):
+    COVER_CHOICES = (
+        ("HARD", "HARD"),
+        ("SOFT", "SOFT"),
+    )
```

Comment:
The first value in choices must be a simple representation (H for HARD and S for SOFT for example). You can get full value by using get_cover_display.
Make something like this:
COVER_CHOICES = (
        ("H", "HARD"),
        ("S", "SOFT"),
    )